src/components/album/Album.js

```diff
@@ -30,14 +32,28 @@ const AlbumTitle = styled('h3')({
 
 function Album({ album, wishListId }) {
   const [openDialog, setOpenDialog] = useState(false);
-
+  const [reviewsUpdated, setReviewsUpdated] = useState(false);
   const handleOpenDialog = () => {
     setOpenDialog(true);
   };
 
   const handleCloseDialog = () => {
     setOpenDialog(false);
+    // Reset the reviewsUpdated state when the dialog is closed
+    setReviewsUpdated(false);
   };
+  // Function to refresh reviews
+  const refreshReviews = () => {
+    // Set reviewsUpdated to true to trigger a re-render of AlbumReviews
+    setReviewsUpdated(true);
+  };
+
+  useEffect(() => {
+    // If reviews are updated, reset the state after rendering AlbumReviews
+    if (reviewsUpdated) {
+      setReviewsUpdated(false);
+    }
+  }, [reviewsUpdated]);
```

Comment:
I'm not sure that I understand the code flow you're aiming to accomplish here

Reply:
@akosasante oh I was trying to add code for this task from PS-78 https://codethedreamteam4-2023.atlassian.net/browse/PS-78  **After submitting, should call the refreshReviews function from parent. In parent component, this should trigger re-fetching the reviews for the page **

Reply:
The parent component can pass in a `refreshReviews` function, for this component to call, but I don't see the need to have a separate state `setReviewsUpdated` that we then follow on every render with `useEffect` to accomplish this.